### DISCUSSION TITLE
Changing colorscheme for lualine when colorscheme of neovim changes

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [nvim-lualine/lualine.nvim#458] [Q&A]
Is it possible to change the colorscheme when default is change? Like I am having a custom theme for my lualine and I want it to change when I switch themes (using telescope) . Like from tokyonight to gruvbox. When i switch colorscheme using telescope, i want the bg,fg to change but keeping the defaults of lualine there

**Comment by shadmansaleh** [comment]
When you use `auto` theme it changes with colorscheme.

**Reply by bryant-the-coder** [reply]
But does it remains all the settings i set?? Like in normal mode i want it to be red.

**Reply by shadmansaleh** [reply]
Colors set though `color` options will be retained but colors from theme will change based on coloracheme.
